sdks/java/core/src/main/java/org/apache/beam/sdk/fn/server/GrpcFnServer.java

```diff
@@ -150,14 +153,12 @@ public Server getServer() {
 
   @Override
   public void close() throws Exception {
+    LOG.info("Shutting down");
```

Comment:
We'll want to remove this debug logging at least, as this will apply to *all* portable Java jobs for each FnAPI service. 

It's not clear we actually want to include the changes in this file, as the previous version was more robust and already calling the various shutdown sequences.

It also has a higher blast radius than the other cleanups you're applying.